recipes/orange3-fairness/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "Orange3-Fairness" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 18f85525bf1a4b1cf6286a92051b6053ad745826311c037cd40482411b95824b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.9,<3.12
+    - setuptools
+  run:
+    - aif360 >=0.5.0
+    - numpy <1.24
+    - orange3 >=3.36
+    - python >=3.9,<3.12
+    - setuptools
```

```suggestion
    - pip
    - python <3.12
    - setuptools
  run:
    - aif360 >=0.5.0
    - numpy <1.24
    - orange3 >=3.32.0
    - python <3.12
```
I tried it on two different spaces and it should work.

* Setuptools is clearly not needed for the run.
* According to the tox file, minimum **Orange3** required minimum version is 3.32.0. I suppose the intention was to have the most recent version installed. However, the conda recipe is supposed to be consistent with the actual requirements from the source code. It also suppresses the inherent flexibility of pure Python packages which is supposed to be the point. https://github.com/biolab/orange3-fairness/blob/8ae2f385962a9ebd911a62ab0e805d3c55f3fcfe/tox.ini#L27 
*Neither Oange3 nor Orange3-fairness require a minimum Python3 version according to Pypi. Aif360 requires 3.7 or higher, but  Conda currently support 3.8 to 3.11, so let's keept that simple.

---

@ZanMervic